packages/menu/src/MenuItem/MenuItem.js

```diff
@@ -6,28 +6,37 @@ import { borderRadius } from 'styled-system'
 import { Check as CheckIcon } from 'pcln-icons'
 
 const MenuButton = styled(Button).attrs(borderRadiusAttrs)`
-  display: flex;
-  justify-content: space-between;
   align-items: center;
+  background-color: ${(props) =>
+    props.selected ? getPaletteColor('light')(props) : getPaletteColor('background.lightest')(props)};
   border: 2px solid transparent;
-  color: ${getPaletteColor('text.dark')};
+  color: ${getPaletteColor('text.base')};
+  display: flex;
+  justify-content: space-between;
   outline: none;
   padding: 12px;
 
   &:hover {
-    background-color: ${getPaletteColor('base')};
+    ${(props) =>
+      props.selected
+        ? `
+        background-color: ${getPaletteColor('light')(props)};
+        color: ${getPaletteColor('dark')(props)};`
+        : `
+        background-color: ${getPaletteColor('background.base')(props)};
+        color: ${getPaletteColor('text.base')(props)};
```

Comment:
I think we still need to allow for custom colours to be set on the button's hover state to be backwards compatible. Also since this is changing from the default `primary.base` on hover it might still be worth making it a breaking/major change.

![Screen Shot 2022-09-26 at 4 46 03 PM](https://user-images.githubusercontent.com/62613356/192386662-02cb4794-6563-4604-9a43-aa8128aa088c.png)
![Screen Shot 2022-09-26 at 4 46 22 PM](https://user-images.githubusercontent.com/62613356/192386670-72db5254-490c-4afe-a795-0fc2f5f5e702.png)

Reply:
Updated to major change! Let me discuss with Wes and other members of design-system team on how we want to handle the custom color hovering.

Reply:
@wesleymartin85 @sdalonzo @pcln-james How should we best support custom hover color while also supporting these new changes? (FYI: I made this a major change)

Reply:
Discussed offline and we're good with only selected colour customization going forward (based on existing implementations getting the in-path update) + also setting it back down to a minor, thanks!

It might be good to split this up into two PRs (move icon changes into a new PR, keep this for the menu update) so that your icon update isn't blocked by testing this.